src/signed-api-fetch/data-fetcher.ts

```diff
@@ -38,23 +38,21 @@ export const stopDataFetcher = () => {
 /**
  * Calls a remote signed data URL.
  * @param url
+ * @param signedDataFetchInterval
  */
-export const callSignedDataApi = async (url: string): Promise<SignedData[] | null> => {
-  const goAxiosCall = await go<Promise<AxiosResponse>, AxiosError>(
-    async () =>
-      axios({
-        method: 'get',
-        timeout: HTTP_SIGNED_DATA_API_ATTEMPT_TIMEOUT - HTTP_SIGNED_DATA_API_HEADROOM / 2,
-        url,
-        headers: {
-          Accept: 'application/json',
-        },
-      }),
-    {
-      attemptTimeoutMs: HTTP_SIGNED_DATA_API_ATTEMPT_TIMEOUT,
-      totalTimeoutMs: HTTP_SIGNED_DATA_API_ATTEMPT_TIMEOUT + HTTP_SIGNED_DATA_API_HEADROOM / 2,
-      retries: 0,
-    }
+export const callSignedDataApi = async (
+  url: string,
+  signedDataFetchIntervalMs: number
+): Promise<SignedData[] | null> => {
+  const goAxiosCall = await go<Promise<AxiosResponse>, AxiosError>(async () =>
+    axios({
+      method: 'get',
+      timeout: signedDataFetchIntervalMs * HTTP_SIGNED_DATA_API_HEADROOM_MULTIPLIER,
```

Comment:
Does this need `Math.ceil` or equivalent? I feel like it could get weird with float rounding errors 🤷

Reply:
There can't by any harm to round it and it's pretty much always a good idea 😅